getting-started/README.md

```diff
@@ -66,33 +68,33 @@ Now we can start the services
 ```
 docker compose up -d
 ```
-It will automatically download the docker image on docker hub:
+It will automatically download the needed docker images from docker hub:
+
 - docker pull opea/chatqna:latest
 - docker pull opea/chatqna-ui:latest
 
-In following cases, you will need to build docker image from source by yourself.
+In the following cases, you will need to build docker image from source by yourself.
 
-1. Failed to download the docker image.
-2. Use the latest or special version.
+- The docker image failed to download.

```

Comment:
Could be a poor internet connection? Should we suggest that they look on docker hub to confirm the image they seek exists there first?